src/d3a/models/area/__init__.py

```diff
@@ -225,9 +233,8 @@ def _update_descendants_strategy_prices(self):
                 self.strategy.event_activate_price()
             for child in self.children:
                 child._update_descendants_strategy_prices()
-        except Exception as e:
-            log.error(f"area._update_descendants_strategy_prices failed. Exception: {e}. "
-                      f"Traceback: {traceback.format_exc()}")
+        except Exception:
+            log.exception("area._update_descendants_strategy_prices failed.")
```

Comment:
Is the traceback also returned for `log.exception` or only the exception type?

Reply:
Yes it will print the full exception traceback